### PR TITLE
Improve Makefile.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,6 +35,11 @@ TEST_FLAGS ?=
 test: toitpkg $(GO_MOCKS)
 	tedi test -v -cover $(TEST_FLAGS) ./...
 
+.PHONY: update-gold
+update-gold: export UPDATE_PKG_GOLD = true
```
Comment (kasperl) [inline Makefile:39]: Is this better than

```
UPDATE_PKG_GOLD=true $(MAKE) test
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: Makefile (hunk 1 above)
Comment (floitsch) [reply Makefile:39]: It does exactly the same.

Comment (floitsch) [reply Makefile:39]: Maybe it's better on Windows, but I'm not sure.

**Pull request merged**